frontend/appflowy_flutter/lib/user/presentation/screens/sign_in_screen/desktop_sign_in_screen.dart

```diff
@@ -0,0 +1,242 @@
+import 'package:appflowy/core/frameless_window.dart';
+import 'package:appflowy/generated/locale_keys.g.dart';
+import 'package:appflowy/user/presentation/screens/sign_in_screen/widgets/widgets.dart';
+import 'package:appflowy/user/presentation/widgets/widgets.dart';
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flowy_infra_ui/flowy_infra_ui.dart';
+import 'package:flutter/material.dart';
+
+class DesktopSignInScreen extends StatelessWidget {
+  const DesktopSignInScreen({super.key, required this.isLoading});
+
+  final bool isLoading;
+
+  @override
+  Widget build(BuildContext context) {
+    const indicatorMinHeight = 4.0;
+    return Scaffold(
+      appBar: const PreferredSize(
+        preferredSize: Size(double.infinity, 60),
+        child: MoveWindowDetector(),
+      ),
+      body: Center(
+        child: AuthFormContainer(
+          children: [
+            FlowyLogoTitle(
+              title: LocaleKeys.welcomeText.tr(),
+              logoSize: const Size(60, 60),
+            ),
+            const VSpace(30),
+            // Email and password. don't support yet.
+            /*
+          ...[
+            const EmailTextField(),
+            const VSpace(5),
+            const PasswordTextField(),
+            const VSpace(20),
+            const LoginButton(),
+            const VSpace(10),
+      
+            const VSpace(10),
+            SignUpPrompt(router: router),
+          ],
+          */
+
+            const SignInAnonymousButton(
+              isMobile: false,
+            ),
+
+            // third-party sign in.
+            const VSpace(20),
+            const _OrDivider(),
+            const VSpace(10),
+            const ThirdPartySignInButtons(
+              isMobile: false,
```

Comment:
Would it be better to remove the `isMobile` parameter and encapsulate the platform logic within the `SignInAnonymousButton`?

Reply:
Good idea! I just updated the code in both `SignInAnonymousButton` and `ThirdPartySignInButtons` for it.